src/MongoDBServiceProvider.php

```diff
@@ -20,6 +21,7 @@
 use MongoDB\Laravel\Eloquent\Model;
 use MongoDB\Laravel\Queue\MongoConnector;
 use RuntimeException;
+use Symfony\Component\HttpFoundation\Session\Storage\Handler\MongoDbSessionHandler;
```

Comment:
Noted that Laravel itself has a dependency on `symfony/http-foundation`. Still, I wonder whether we should either add it to `require` or `require-dev`, and/or add a `conflict` entry to ensure we're getting a version of the session handler that we expect instead of relying on the transitive dependency.

Reply:
In fact, it's best to be explicit about this dependency. Laravel 10 requires symfony 6.4 while laravel 11 requires symfony 7.